tests/integration/test_amax_amin.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2021-2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+from legate.core import LEGATE_MAX_DIM
+
+import cunumeric as num
+
+FUNCS = ("amax", "amin")
+
+
+@pytest.mark.parametrize("keepdims", [True, False])
+@pytest.mark.parametrize("ndim", range(LEGATE_MAX_DIM + 1))
+@pytest.mark.parametrize("func_name", FUNCS)
+def test_basic(func_name, ndim, keepdims):
+    shape = (5,) * ndim
+    in_np = np.random.random(shape)
+    in_num = num.array(in_np)
+
+    func_np = getattr(np, func_name)
+    func_num = getattr(num, func_name)
+
+    assert np.array_equal(
+        func_np(in_np, keepdims=keepdims),
+        func_num(in_num, keepdims=keepdims),
+    )
+
+    for val in (-2, 0.5, 2):
+        assert np.array_equal(
+            func_np(in_np, keepdims=keepdims, initial=val),
+            func_num(in_num, keepdims=keepdims, initial=val),
+        )
```

Comment:
What do you think about handling `initial` as a parameter so that the test can follow the AAA (arrange - act - assert) pattern that most of the tests already follow? If you agree, please apply this wherever else it fits. 
```suggestion
@pytest.mark.parametrize("initial", (None, -2, 0, 0.5, 2))
@pytest.mark.parametrize("func_name", FUNCS)
def test_basic(func_name, initial, ndim, keepdims):
    # arrange
    shape = (5,) * ndim
    in_np = np.random.random(shape)
    in_num = num.array(in_np)

    func_np = getattr(np, func_name)
    func_num = getattr(num, func_name)
    
    kw = {} if initial is None else dict(initial=val)
    
    # act
    res_np = func_np(in_np, keepdims=keepdims, **kw)
    res_num = func_num(in_num, keepdims=keepdims, **kw)

    # assert
    assert np.array_equal(res_np, res_num)
```

Reply:
Fixed. Thanks.